classes/06class/exercises/c06-serverless01/ANSWER.md

```diff
@@ -0,0 +1,15 @@
+# c06-serverless01
+
+## Make
```

Comment:
Is this supposed to have the `Make` title?

Reply:
Fixed.